examples/file-feed/complete/config/webpack.config.dev.js

```diff
@@ -0,0 +1,134 @@
+const getClientEnvironment = require('./env')
+const paths = require('./paths')
+
+// @remove-on-eject-begin
+const path = require('path')
+// @remove-on-eject-end
+
+const publicPath = '/'
+const publicUrl = ''
+const env = getClientEnvironment(publicUrl)
+
+module.exports = {
+  devtool: 'cheap-module-source-map',
+  entry: [
+    require.resolve('react-dev-utils/webpackHotDevClient'),
+    require.resolve('./polyfills'),
+    paths.appIndexJs
+  ],
+  resolve: {
+    fallback: paths.nodePaths,
+    extensions: ['', '.js', '.json', '.jsx'],
+    alias: {
+      'react-native': 'react-native-web',
+      'zlib': 'browserify-zlib-next'
+    }
+  },
+  // @remove-on-eject-begin
+  resolveLoader: {
+    root: paths.ownNodeModules,
+    moduleTemplates: ['*-loader']
+  },
+  output: {
+    path: paths.appBuild,
+    pathinfo: true,
+    filename: 'static/js/bundle.js',
+    publicPath: publicPath
+  },
+  // @remove-on-eject-end
+  module: {
+    preLoaders: [
+      {
+        test: /\.(js|jsx)$/,
+        loader: 'eslint',
+        include: paths.appSrc
+      }
+    ],
+    loaders: [
+      {
+        exclude: [
+          /\.html$/,
+          /\.(js|jsx)$/,
+          /\.css$/,
+          /\.json$/,
+          /\.svg$/
+        ],
+        loader: 'url',
+        query: {
+          limit: 10000,
+          name: 'static/media/[name].[hash:8].[ext]'
+        }
+      },
+      // Process JS with Babel.
+      {
+        test: /\.(js|jsx)$/,
+        include: paths.appSrc,
+        loader: 'babel',
+        query: {
+          // @remove-on-eject-begin
+          babelrc: false,
+          presets: [require.resolve('babel-preset-react-app')],
+          // @remove-on-eject-end
+          cacheDirectory: false
+        }
+      },
+      {
+        test: /\.css$/,
+        loader: 'style!css?importLoaders=1!postcss'
+      },
+      {
+        test: /\.json$/,
+        loader: 'json'
+      },
+      {
+        test: /\.svg$/,
+        loader: 'file',
+        query: {
+          name: 'static/media/[name].[hash:8].[ext]'
+        }
+      }
+    ]
+  },
+  // @remove-on-eject-begin
+  eslint: {
+    configFile: path.join(__dirname, '../.eslintrc'),
+    useEslintrc: false
+  },
+  // @remove-on-eject-end
+  postcss: function () {
+    return [
+      autoprefixer({
+        browsers: [
+          '>1%',
+          'last 4 versions',
+          'Firefox ESR',
+          'not ie < 9' // React doesn't support IE8 anyway
+        ]
+      })
+    ]
+  },
+  plugins: [
+    new InterpolateHtmlPlugin({
+      PUBLIC_URL: publicUrl
+    }),
+    new HtmlWebpackPlugin({
+      inject: true,
+      template: paths.appHtml
+    }),
+    new webpack.DefinePlugin(env),
+    new webpack.HotModuleReplacementPlugin(),
+    new CaseSensitivePathsPlugin(),
+    new WatchMissingNodeModulesPlugin(paths.appNodeModules)
+  ],
+  node: {
+    fs: 'empty',
+    net: 'empty',
+    tls: 'empty'
+  }
+}
```

Comment:
Yeah, we need all this configuration. See my long comment below as to why.

Short: the webpack config file here is the create-react-app/react-script config file with the only difference that we add the zlib shim to it.